.tool-versions

```diff
@@ -0,0 +1 @@
+dotnet-core 6.0.406
```

Comment:
QQ, we are usually not pushing this in public repos, are we standardizing that?

Reply:
Yes, we'd like to standardise and make it simple for all internal contributors, and hopefully, external contributors might find it helpful as well.